AUTHORS.md

```diff
@@ -8,20 +8,22 @@ Hawaiʻi at Mānoa.
 The following people have contributed code and/or documentation to the project
 (alphabetically by name) and are considered to be "PyGMT Developers":
```

Comment:
```suggestion
(alphabetically by first name) and are considered to be "PyGMT Developers":
```

Reply:
This is deliberately just 'name' (or full name) because some cultures don't have first/last names (see https://github.com/GenericMappingTools/pygmt/pull/857#discussion_r574098862).

Reply:
Ah, that's a good point!
Then just leave it as is 🙂.